guidelines/2.4.3.md

```diff
@@ -95,6 +95,51 @@ This section needs a review and more content. [Contribute via Github](https://gi
 
 ## Guidance for Android
 
+The best practices for grouping and ordering elements in an accessible manner can be summarized as follows:
+
+* The view hierarchy order and on-screen positioning determine grouping and ordering of text in spoken feedback.
+
+* You should group non-focusable items in a focusable container to have them read as a single item.
+
+* To logically group related items, it is sometimes necessary to create nested **ViewGroup** elements (eg: a wrapping RelativeLayout for a label and its value)
+
+```xml
+<LinearLayout
+    ...
+    orientation="vertical">
+
+    <RelativeLayout
+        ...
+        android:focusable="true">
+
+        <TextView
+          ...  />
+        <TextView
+          ...  />
+    </RelativeLayout>
+
+    <RelativeLayout
+        ...
+        android:focusable="true">
+
+        <TextView
+          ... />
+        <TextView
+          ...  />
+    </RelativeLayout>
+
+    <RelativeLayout
+        ...
+        android:focusable="true">
+
+        <TextView
+          ...  />
+        <TextView
+          ...  />
+    </RelativeLayout>
+</LinearLayout>
+```
+
 This section needs a review and more content. [Contribute via Github](https://github.com/theappbusiness/accessibility-guidelines/) or [email](mailto:jeanfrancois@theappbusiness.com).
```

Comment:
Side bar: we probably need to go through and remove/update stuff like this, at least point it to another email address that still exists

Reply:
I thought as well. Since we have a bench task for updating from JF I thought this would be tackled later on.

Reply:
np!